src/main/java/frc/team4909/robot/subsystems/elevator/commands/SetElevatorPosition.java

```diff
@@ -1,19 +1,28 @@
 package frc.team4909.robot.subsystems.elevator.commands;
 
+import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.InstantCommand;
 import frc.team4909.robot.subsystems.elevator.ElevatorSubsystem;
 import frc.team4909.robot.Robot;
 
 public class SetElevatorPosition extends InstantCommand {
-    private int setpoint;
+    private int setpoint, threshold;
 
-    public SetElevatorPosition(int setpoint) {
-        this.setpoint = setpoint;       
+    public SetElevatorPosition(int setpoint, int threshold) {
+        this.setpoint = setpoint;  
+        this.threshold = threshold;     
         requires(Robot.elevatorSubsystem);
     }
 
     @Override
     public void initialize() {
```

Comment:
why in initialize? theoretically it shouldn't matter but I'd do it in execute to avoid being overwritten and to avoid any timeout issues that may be hidden